packages/wonder-blocks-timing/src/util/timeout.ts

```diff
@@ -15,7 +15,7 @@ import type {ITimeout, SchedulePolicy, ClearPolicy} from "./types";
  * @implements {ITimeout}
  */
 export default class Timeout implements ITimeout {
-    _timeoutId: number | null | undefined;
+    _timeoutId: ReturnType<typeof setTimeout> | null | undefined;
```

Comment:
**note**: This was a tricky one. The global `setTimeout` has a return type of `NodeJS.Timeout`. This is what is likely correct for in tests where we're not in a JSDOM, but the `window.setTimeout` call that will run in browsers is typed to return a number. 

The easy fix to satisfy TS without a functional change is what we have here.